skbuild/resources/cmake/FindPythonExtensions.cmake

```
@@ -254,19 +254,27 @@ endif()
 include(targetLinkLibrariesWithDynamicLookup)
 
 set(_command "
-import distutils.sysconfig
+import sys
+at_least_python_3_10 = sys.version_info[:2] >= (3, 10)
+if at_least_python_3_10:
```

```suggestion
if sys.version_info >= (3, 10):
```

This is an idiom already.